how.md

```diff
@@ -4,3 +4,14 @@ This is a petri dish for random thoughts we have on what principles or practices
 
 To start, each idea is in a H2, here... as we mature ideas, we'll re-home them.
 
+## Authentic Culture Emerges
+
+- Most organizations' "values" are hatched by "leadership" and then plastered all over the group's space like commandments.
+- Authentic values are what we observe sparking between people within the group... and we fan the flames by giving the experience some words.
```

Comment:
>  Authentic values are what we observe sparking between people within the group... and we fan the flames by giving the experience some words.

Well said. I strongly agree with this.

I assume there's also some filtering aspect here where the group may need to determine which observed values the organization 'wants' to encourage. For example, if there is an observed value of "when we disagree, we prefer to let the loudest voice in the room decide", that would probably not be a value the organization would want to fan the flames of by naming and encouraging.

🤔  How does a self-organizing group decide which observed cultural values are worth encouraging and which are not? What if there's internal disagreement over whether some observed value is 'good'?

I'm not sure if I'm going anywhere useful with this, but just sorta thinking out loud in case in sparks something in someone else.